### PR TITLE
Increase default font size of pl-python-variable dataframes

### PR DESCRIPTION
**Pull request by ffund**
Previously, the font size of a `pandas` dataframe displayed using a `pl-python-variable` was smaller than the surrounding text.

### CODE REVIEW

**Comment by ffund** [comment]
Before:

image

After:

image

File: elements/pl-python-variable/pl-python-variable.css [hunk 1]
```diff
@@ -2,7 +2,7 @@
     display: block;
     overflow-x: scroll;
     font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
-    font-size: 12px;
+    font-size: 1rem;
```
Comment (nwalters512) [inline elements/pl-python-variable/pl-python-variable.css:5]: Could we just delete the `font-size` property entirely and use the Bootstrap defaults?

Comment (mwest1066) [reply elements/pl-python-variable/pl-python-variable.css:5]: Good thinking! @ffund does it look sensible if we just don't set a `font-size` at all? I'm also tempted to remove `font-family` unless there's a good reason we are specifying it?

Comment (ffund) [reply elements/pl-python-variable/pl-python-variable.css:5]: Sounds fine. if we do that, we should also remove `font-family` for the 'dimensions' line at the bottom.

**Pull request merged**